lib/fluent/plugin/slack_client.rb

```diff
@@ -114,7 +114,8 @@ def post_message(params = {}, opts = {})
       private
 
       def encode_body(params = {})
-        params.to_json
+        # https://api.slack.com/docs/formatting
+        params.to_json.gsub(/&/, '&amp;').gsub(/</, '&lt;').gsub(/>/, '&gt;')
```

Comment:
Slack document says
```
& replaced with &amp;
< replaced with &lt;
> replaced with &gt;
```
but `"` is not need to escape.

If `"` converted to `&quot;` (by CGI.escapeHTML)
```
[info]: out_slack: post_message {:channel=>"#sandbox", :text=>"\"xxx\"\n", :mrkdwn=>true, :link_names=>true}
```

displayed in Slack as below.
```
&quot;xxx&quot; 
```